services-toolkit/how-to-guides/configure-dynamic-provisioning-rds.hbs.md

```diff
@@ -0,0 +1,208 @@
+# Configure Dynamic Provisioning of AWS RDS Service Instances
+
+## <a id="stk-dynamic-provisioning-rds-about"></a> About
+
+This document details the steps required to setup dynamic provisioning of AWS RDS service instances. If you are not already familiar with dynamic provisioning in Tanzu Application Platform, it is recommended to first run through the tutorial [Setup Dynamic Provisioning of Service Instances](../tutorials/setup-dynamic-provisioning.hbs.md). The learnings from that tutorial will be beneficial to understanding the steps presented here.
+
+## <a id="stk-dynamic-provisioning-rds-pre-reqs"></a> Pre-requisites
+
+* Access to a TAP cluster (version >= 1.5.0)
+* The `tanzu services` CLI plug-in (version >= 0.6.0)
+* Access to AWS
+
+## <a id="stk-dynamic-provisioning-rds-step-1"></a> Step 1: Install the AWS Provider for Crossplane
+
+The first step is to install the AWS `Provider` for Crossplane. There are two variants of the `Provider` - [crossplane-contrib/provider-aws](https://marketplace.upbound.io/providers/crossplane-contrib/provider-aws/) and [upbound/provider-aws](https://marketplace.upbound.io/providers/upbound/provider-aws/). VMware recommends you install the official Upbound variant. Follow the official documentation [here](https://marketplace.upbound.io/providers/upbound/provider-aws/latest/docs/quickstart) to install the `Provider` and to create a corresponding `ProviderConfig`. 
```

Comment:
*question* Do we know if they both contain the same APIs so it doesn't matter from the composition perspective?

Reply:
Good question I'm not sure - I'll check that out.